views/route_segments/route_segments_show.html

```diff
@@ -28,7 +29,7 @@ <h2 class="{{ IconFor `route-segment` }}">{{ .Name }}</h2>
           <div class="inner-form">{{ template "route_segment_details" . }}</div>
         </div>
         <div class="basis-1/2 2xl:basis-1/3">
-          <div class="inner-form">
+          <div class="inner-form max-h-[640px] overflow-y-auto">
```

Comment:
The map on the left has the following properties:

`h-[300px] sm:h-[400px] md:h-[600px] print:h-[600px]`

This means that smaller screens will show a smaller map. Should we add this for the breakdown too? (Adding a bit extra because of the buttons on the bottom)